build/release-patcher-Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM golang:1.24
+
+RUN apt-get update
```

Comment:
[nit] We could massage this, there are some common tweaks about using apt-get in dockerfiles, but since this is really niche file for our tooling purposes (i.e. something to get rid of soon, please), let's roll with it.